desktop/apps/categories3/components/Gene.js

```diff
@@ -1,16 +1,31 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import styled from 'styled-components'
 
 const propTypes = {
   id: PropTypes.string.isRequired,
   name: PropTypes.string.isRequired
 }
 
+const GeneItem = styled.li`
+  font-size: 20px;
+  line-height: 1.33em;
```

Comment:
Would be nice to eventually get our type system working here, but not a blocker. See: https://github.com/artsy/force/blob/master/desktop/components/stylus_lib/typography.styl

Reply:
In Reaction's publishing folder, we export these utility functions: https://github.com/artsy/reaction/blob/master/src/components/publishing/fonts.tsx

and then it gets used like this:
```
import Fonts from "./fonts"

const Text = styled.div`
   ${Fonts.garamond("s20")}
`
```
There's also the non-publishing Text component in Reaction here with Typescript: https://github.com/artsy/reaction/blob/master/src/components/text.tsx

Reply:
@kanaabe - Don't you think we should just use Reactions utilities directly in Force React code? Since Reaction is already a dependency and everything is publicly available its as easy as an `import`. (I'm about to start on some Auctions / Reaction stuff so i'm trying to also think about how best to go about it, too)

Reply:
@damassi 👍  Yea I like that! I personally prefer to use the utility function style because I think it makes the render function cleaner, but the right one to use would be the component one: https://github.com/artsy/reaction/blob/master/src/components/text.tsx. I think this one maps to our style guide. 

There's also [this](https://github.com/artsy/reaction/blob/master/src/assets/colors.tsx) for colors.

Reply:
Cool, so I've pulled in shared font and color specs from Reaction as follows:

```js
import colors from '@artsy/reaction-force/dist/assets/colors'
import { primary, secondary } from '@artsy/reaction-force/dist/assets/fonts'
```

And then sprinkled them through my styled components, as necessary.

Need to take a look at the next step of getting type specs beyond just than basic font-family stuff -- will study that `Text` component from Reaction, but might merge this in the meantime if all looks good.

Reply:
Yeah go for it! I see what you did with the `primary.style` -- it's a good start. I saw that using `<Text/>` wraps your children in a `<p>`, so we can't use it right away for this PR anyway.